.github/workflows/build_and_test.yml

```diff
@@ -164,9 +165,12 @@ jobs:
     name: test unit
     strategy:
       fail-fast: false
+      matrix:
+        node: [18, 20] # TODO: use env var like [env.NODE_MAINTENANCE_VERSION, env.NODE_LTS_VERSION]
```

Comment:
Looks like we would want to use [GitHub Action Variables](https://docs.github.com/en/actions/learn-github-actions/contexts#vars-context) here:

![image](https://github.com/vercel/next.js/assets/18369201/4ecf3115-4361-45a0-b16a-8516a5a4b644)